android/app/src/main/python/electroncash_gui/android/strings.py

```diff
@@ -32,6 +32,7 @@ def _(s):
 _("ID")
 _("Import addresses or private keys")
 _("Invalid address")
+_("Load unbroadcasted transaction")
```

Comment:
I think "unbroadcasted" is redundant and can be removed.

Reply:
removed